### DISCUSSION TITLE
能否隐藏运行时的控制台

### DISCUSSION THREAD

**Discussion by gylang** [messense/aliyundrive-webdav#234] [Q&A] [answered]
能否隐藏运行时的控制台

**Comment by messense** [comment, marked as answer]
参考 https://github.com/shadowsocks/shadowsocks-rust/issues/333

**Reply by yfdyh000** [reply]
他想要静默运行（-silent 或者服务），或者是窗口隐藏到托盘。

**Reply by messense** [reply]
本来就是个 console 应用，隐藏没必要。

**Reply by yfdyh000** [reply]
有人用来开机自启、当服务器长期运行吧，有个窗口会碍眼。可以用第三方软件隐藏窗口。
顺便问一下，默认监听0.0.0.0端口且弱密码是否有访问风险？包括v4和v6的公网/内网访问。软件不支持显示访问流量，默认也没日志。

**Reply by messense** [reply]
设计是家庭内网运行的，如果服务器上运行，应自行配置监听地址端口和强密码。
